components/markdown/markdown.tsx

```diff
@@ -133,9 +149,11 @@ export default class Markdown extends React.PureComponent<Props> {
             team: this.props.team,
             minimumHashtagLength: this.props.minimumHashtagLength,
             managedResourcePaths: this.props.managedResourcePaths,
+            editedAt,
         }, this.props.options);
 
-        const htmlFormattedText = formatText(this.props.message, options, this.props.emojiMap);
+        const htmlFormattedText = formatText(message, options, this.props.emojiMap, postId);
```

Comment:
Perhaps the postId should also be added to options?

Reply:
good idea ... it was messing with the types last time i tried that, but it is a better solution, so i will try once more!